src/components/atoms/Skeleton/Title.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components'
+import { LineSkeleton, LineSkeletonProps } from './Line'
+
+// I N T E R F A C E S
+export interface TitleSkeletonProps extends LineSkeletonProps {
+  title?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
+  className?: string
+  style?: React.CSSProperties
+}
```

Comment:
`LineSkeletonProps` already contains `className` and `style` so you can reduce to:

```suggestion
// I N T E R F A C E S
export interface TitleSkeletonProps extends LineSkeletonProps {
  title?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
}
```